packages/react-instantsearch-core/src/core/createInstantSearchManager.js

```diff
@@ -271,14 +271,18 @@ export default function createInstantSearchManager({
     search();
   }
 
-  function onSearchForFacetValues({ facetName, query, maxFacetHits }) {
+  function onSearchForFacetValues({ facetName, query, maxFacetHits = 10 }) {
+    // The values 1, 100 are the min / max values that the engine accepts.
+    // see: https://www.algolia.com/doc/api-reference/api-parameters/maxFacetHits
+    const maxFacetHitsWithinRange = Math.max(1, Math.min(maxFacetHits, 100));
```

Comment:
Do we have this issue as well in InstantSearch.js?

Reply:
We have kinda the same issue, but it's less visible because we don't use the `showMore` option to conditionally pick the value for `maxFacetHits`.